### PR TITLE
Mod05 demo2

### PR DESCRIPTION
**Pull request by shalevz**

### CODE REVIEW

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (AviAvni) [inline AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj]: change to 2.1

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.3" />
```
Comment (AviAvni) [inline AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj]: update to 2.1 packages

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj.user [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (AviAvni) [inline AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj.user]: delete this file

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/Properties/PublishProfiles/CustomProfile.pubxml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AviAvni) [inline AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/Properties/PublishProfiles/CustomProfile.pubxml]: delete this file

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/Properties/PublishProfiles/CustomProfile.pubxml.user [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AviAvni) [inline AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/Properties/PublishProfiles/CustomProfile.pubxml.user]: delete this file

File: Instructions/20487D_MOD05_DEMO.md [hunk 6]
```diff
@@ -36,6 +36,37 @@
...
+12. In the **Open Project** dialog box, browse to **[Repository Root]\Allfiles\Mod05\DemoFiles\Host in IIS**, click **BlueYonder.Hotels.Service.sln**, and then click **Open**.
+13. Right-click on **BlueYonder.Hotels.Service** and then click **Publish**.
+14. In the **Publish** blade, choose **IIS, FTP, etc** and then click **Create Profile**.
+15. In the **Publish** window, type **localhost** in the **Server** box.
+16. Type **HotelsSite** in the **Site name** box and then press **Next**.
+17. In the **Settings** pane, choose **Self-Contained** in the **Deployment Mode** combo-box.
+18. Choose **win-x64** in the **Target Runtime** combo-bx and then click **Save**.
+19. In the **Publish** blade , click **Publish**.
+20. Wait until the process will finish.
+21. Open the browser and browse to **localhost:8080/api/Values**. Check that you are getting a good response.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: how good response look like?

File: Instructions/20487D_MOD05_DEMO.md [hunk 7]
```diff
@@ -36,6 +36,37 @@
...
+22. Close the Browser.
+23. Close **Visual Studio**.
```
Comment (AviAvni) [inline Instructions/20487D_MOD05_DEMO.md]: I think this 2 steps are redundant. the user know when he want to close the browser and visual studio. also our  QA don't need to check that closing this apps is working

File: AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj (hunk 2 above)
Comment (AviAvni) [reply AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj]: Not 2.01 you need 2.1.0

Comment (shalevz) [reply AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj]: yes its doing the same

Comment (shalevz) [reply AllFiles/Mod05/DemoFiles/Host in IIS/BlueYonder.Hotels.Service/BlueYonder.Hotels.Service.csproj]: done

**Pull request merged**